packages/core/src/flushPolicies/background-flush-policy.ts

```diff
@@ -18,7 +18,7 @@ export class BackgroundFlushPolicy extends FlushPolicyBase {
       'change',
       (nextAppState) => {
         if (
-          this.appState === 'active' &&
+          (this.appState === 'active' || 'unknown') &&
```

Comment:
If left as right now it would use the value `unknown` and always be true. To do the intended check if would have to be as in: 

```suggestion
          (this.appState === 'active' || this.appState === 'unknown') &&
```